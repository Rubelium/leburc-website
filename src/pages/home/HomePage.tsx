import Body from "../../feature/body/Body";
import Header from "../../feature/header/Header";

const HomePage = () => {
  return (
    <div className="home-page">
      <Header />
      <Body />
    </div>
  );
};

export default HomePage;
