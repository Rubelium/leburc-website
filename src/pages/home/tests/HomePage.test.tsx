import { render } from "@testing-library/react";
import HomePage from "../HomePage";

describe("it should render", () => {
  test("the snapshot of the <HomePage /> component", () => {
    const component = render(<HomePage />);

    expect(component).toMatchSnapshot();
  });
});
