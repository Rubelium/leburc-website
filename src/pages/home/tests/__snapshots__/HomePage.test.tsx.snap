// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the snapshot of the <HomePage /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home-page"
      >
        <header
          class="app-header"
        >
          <img
            alt="F2T4 Logo"
            class="app-logo"
            src="logoF2t4BlackWhiteCircleFilled.png"
          />
          <img
            alt="F2T4 Title"
            class="app-title"
            src="titleF2t4White.png"
          />
          <br />
          <h1
            style="display: none;"
          >
            Floor to the Four - Floor⁴
          </h1>
          <h2
            style="font-size: 1.5rem;"
          >
            SERVING THE PRIMEST CUTS OF MODERN TECHNO AND HOUSE
          </h2>
        </header>
        <div
          class="app-body-section"
        >
          <div
            class="spinner-border text-dark"
            role="status"
          >
            <span
              class="visually-hidden"
            >
              Loading...
            </span>
          </div>
        </div>
        <div
          class="app-body-section-alt"
        >
          <div
            class="spinner-border text-light"
            role="status"
          >
            <span
              class="visually-hidden"
            >
              Loading...
            </span>
          </div>
        </div>
        <div
          class="app-body-section"
        >
          <div
            class="spinner-border text-dark"
            role="status"
          >
            <span
              class="visually-hidden"
            >
              Loading...
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home-page"
    >
      <header
        class="app-header"
      >
        <img
          alt="F2T4 Logo"
          class="app-logo"
          src="logoF2t4BlackWhiteCircleFilled.png"
        />
        <img
          alt="F2T4 Title"
          class="app-title"
          src="titleF2t4White.png"
        />
        <br />
        <h1
          style="display: none;"
        >
          Floor to the Four - Floor⁴
        </h1>
        <h2
          style="font-size: 1.5rem;"
        >
          SERVING THE PRIMEST CUTS OF MODERN TECHNO AND HOUSE
        </h2>
      </header>
      <div
        class="app-body-section"
      >
        <div
          class="spinner-border text-dark"
          role="status"
        >
          <span
            class="visually-hidden"
          >
            Loading...
          </span>
        </div>
      </div>
      <div
        class="app-body-section-alt"
      >
        <div
          class="spinner-border text-light"
          role="status"
        >
          <span
            class="visually-hidden"
          >
            Loading...
          </span>
        </div>
      </div>
      <div
        class="app-body-section"
      >
        <div
          class="spinner-border text-dark"
          role="status"
        >
          <span
            class="visually-hidden"
          >
            Loading...
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
